### ISSUE TITLE
CoreML convert tf's ArgMax output wrongly.

### ISSUE THREAD

**Issue by FrozenGene** [tf-coreml/tf-coreml#188]
See tensorflow's output type:

![image](https://user-images.githubusercontent.com/7287321/40572705-7bf5abf8-60e5-11e8-837a-34d771493e87.png)

But CoreML's output type:

![image](https://user-images.githubusercontent.com/7287321/40572710-9f3d49b8-60e5-11e8-940f-641bdb8e7650.png)

Anyway we can specify the output's type to workaroud before we solve it? Thanks.

**Comment by aseemw** [comment]
Yes that is a good point. This should be added to the converter. Let us keep this issue open to track this. 
(Do you mind renaming this issue to something like "set input/output datatype correctly". This problem is not argmax related so the current title is a bit mis-leading). 

In the meantime, you can set the datatype of input/output like this: 

```
from coremltools.proto import FeatureTypes_pb2 as ft
feature_ = spec.description.output[0]
if feature_.type.HasField('multiArrayType'):
    feature_.type.multiArrayType.dataType = ft.ArrayFeatureType.INT32
```

**Comment by FrozenGene** [comment]
Thanks. Have modified the title.
